src/Extensions/ServiceFabricGuest.Abstractions/internal/ServiceFabricClientWrapper.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Options;
+using Microsoft.ServiceFabric.Client;
+
+namespace Microsoft.Omex.Extensions.ServiceFabricGuest.Abstractions
+{
+	internal class ServiceFabricClientWrapper : IServiceFabricClientWrapper
+	{
+		private IServiceFabricClient? m_client;
+
+		private Uri m_clusterEndpoint;
+
+		public ServiceFabricClientWrapper(IOptions<ServiceFabricRestClientOptions> options)
+		{
+			m_clusterEndpoint = new(options.Value.ClusterEndpoint);
+		}
+
+		public async Task<IServiceFabricClient> GetAsync()
```

Comment:
do you think the naming here could be more descriptive (GetAsync)? GetClientAsync maybe?

Reply:
Actually it's probably fine as is

Reply:
To be honest I am not sure if it will be ok there - I mean, the information that this is store for client is already noted in name of the class. `ServiceFabricClientWrapper.Get()` for me means that we extracts stored value and returns it (also it is mentioned in IServiceFabricClient docs). So to sum up - I am not sure if it will be more descriptive but if you think so, I am open to change

Reply:
No it's fine to leave it :)